src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -296,18 +155,18 @@ bool AdaptDispatch::CursorNextLine(const size_t distance)
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::CursorPrevLine(const size_t distance)
 {
-    return _CursorMovement(CursorDirection::PrevLine, distance);
+    return _CursorMovePosition(Offset::Backward(distance), Offset::Absolute(1), true);
```

Comment:
Can you leave the note here that we're supposed to unconditionally move to the left edge up one line? That was a comment in the previous code that you deleted but isn't here.

I was momentarily confused about that (as I mentally thought previous line should go up one line and to the right most column like a backspace that's allowed to wrap up a line) until I went and read the removed code/comments in `_CursorMovement`.

Reply:
I've not sure what you mean? The comment is still here:
https://github.com/microsoft/terminal/blob/d6c4f35cf60a239cb1b8b7b7cc5796b06f78a884/src/terminal/adapter/adaptDispatch.cpp#L150-L151

Reply:
My mistake, sorry.